charts/datadog/requirements.yaml

```diff
@@ -2,7 +2,7 @@ dependencies:
   - name: datadog-crds
     version: 1.0.1
     repository: https://helm.datadoghq.com
-    condition: clusterAgent.metricsProvider.useDatadogMetrics
+    condition: datadog.autoscaling.workload.enabled,clusterAgent.metricsProvider.useDatadogMetrics
```

Comment:
dummy question: `condition` is using "OR" operator between the fields?

Reply:
Unfortunately no, it's fallback mechanism: it takes the first field that **exists** (not that says `true`), hence it relies on the fact that `datadog.autoscaling.workload.enabled` does not exist in our default `values.yaml`.